src/account/account.cairo

```diff
@@ -154,24 +166,14 @@ mod Account {
         }
     }
 
-    #[external(v0)]
-    fn __validate_deploy__(
-        self: @ContractState,
-        class_hash: felt252,
-        contract_address_salt: felt252,
-        _public_key: felt252
-    ) -> felt252 {
-        self.validate_transaction()
-    }
-
     //
     // Internal
     //
 
     #[generate_trait]
     impl InternalImpl of InternalTrait {
         fn initializer(ref self: ContractState, _public_key: felt252) {
-            let mut unsafe_state = src5_state();
+            let mut unsafe_state = SRC5::unsafe_new_contract_state();
```

Comment:
Should we keep this? It should be removed when migrating to component anyway, buy I think src5_state looks better.

Reply:
Agreed! I think this got mixed up in the merge/release. Good catch